features/unit/v2algodclient_paths.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Algod REST API v2
+  Background:
+    Given mock server recording request paths
+  # SendRaw, Status, Supply, SuggestedParams, and PendingTransactionsInformation omitted - the path never mutates, they're constant
+
+  Scenario Outline: Shutdown
+    When we make a Shutdown call with timeout <timeout>
+    Then expect the path used to be "<path>"
+    Examples:
+      |path   | timeout |
+      |TODO   | 0 |
+
+  Scenario Outline: Register Participation Keys
+    When we make a Register Participation Keys call against account "<account>" fee <fee> dilution <dilution> lastvalidround <lastvalid> and nowait "<nowait>"
+    Then expect the path used to be "<path>"
+    Examples:
+      |path   | account | fee | dilution | lastvalid | nowait|
+      |TODO   | TODO    | 0   | 0        | 0         | false |
+
+  Scenario Outline: Pending Transaction Information
```

Comment:
Should this be two separate scenarios, one for pending transaction information by txid, and one for pending transactions?

Reply:
Yes, good find! Fixing. (x2)